src/classes/os-utils.ts

```diff
@@ -53,40 +53,19 @@ export default class OsUtils {
     }
   }
 
-    /**
+  /**
    * Re-writes user config using previous user config as basis
    * 
    * @param os is the OS that is being switched to, if supported
    */
   private static async swapOs (os : string) {
-    const userConfig : UserConfigJson = await OsUtils.readUserConfig();
+    const userConfig : UserConfigJson = await userConfigUtils.readUserConfig();
```

Comment:
Class names should be captitalized
```suggestion
    const userConfig : UserConfigJson = await UserConfigUtils.readUserConfig();
```